src/enums/filingStatus.ts

```diff
@@ -6,5 +6,6 @@ export enum FilingStatus {
   NEW = 'NEW',
   PAID = 'PAID',
   PENDING = 'PENDING',
+  PENDING_ALTERATION = 'PENDING_ALTERATION',
```

Comment:
I assume we will have this (for background alteration processing). If not, no harm done.